docs/topics/wasm/wasm-get-started.md

```diff
@@ -1,39 +1,38 @@
 [//]: # (title: Get started with Kotlin/Wasm and Compose Multiplatform)
 
-> Kotlin/Wasm is in [Alpha](components-stability.md). It may be changed at any time.
-> 
-> [Join the Kotlin/Wasm community.](https://slack-chats.kotlinlang.org/c/webassembly)
->
-{style="note"}
+<primary-label ref="beta"/> 
 
 This tutorial demonstrates how to run a [Compose Multiplatform](https://www.jetbrains.com/lp/compose-multiplatform/) 
-app with [Kotlin/Wasm](wasm-overview.md) in IntelliJ IDEA, and generate artifacts to publish as a site on [GitHub pages](https://pages.github.com/).
+app with [](wasm-overview.md) in IntelliJ IDEA and generate artifacts to publish on a site.
 
-## Before you start
+## Create a project
 
-Create a project using the Kotlin Multiplatform wizard:
+1. [Set up your environment for Kotlin Multiplatform development](https://www.jetbrains.com/help/kotlin-multiplatform-dev/quickstart.html#set-up-the-environment).
+2. In IntelliJ IDEA, select **File | New | Project**.
+3. In the panel on the left, select **Kotlin Multiplatform**.
 
-1. Open the [Kotlin Multiplatform wizard](https://kmp.jetbrains.com/#newProject).
-2. On the **New Project** tab, change the project name and ID to your preference. In this tutorial, we set the name to "WasmDemo" and the ID to "wasm.project.demo".
-
-   > These are the name and ID of the project directory. You can also leave them as they are.
+   > If you're not using the Kotlin Multiplatform IDE plugin, you can generate the same project using the [KMP web wizard](https://kmp.jetbrains.com/?android=true&ios=true&iosui=compose&desktop=true&web=true&includeTests=true).
    >
-   {style="tip"}
+   {style="note"}
 
-3. Select the **Web** option. Make sure that no other options are selected.
-4. Click the **Download** button and unpack the resulting archive.
+4. Specify the following fields in the **New Project** window:
 
-   ![Kotlin Multiplatform wizard](wasm-compose-web-wizard.png){width=400}
+   * **Name:** WasmDemo
+   * **Group:** wasm.project.demo
+   * **Artifact:** wasmdemo
 
-## Open the project in IntelliJ IDEA
+   > If you use the web wizard, specify "WasmDemo" as Project Name and "wasm.project.demo" as Project ID.
+   >
+   {style="note"}
 
-1. Download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/).
-2. On the Welcome screen of IntelliJ IDEA, click **Open** or select **File | Open** in the menu bar.
-3. Navigate to the unpacked "WasmDemo" folder and click **Open**.
+5. Select the **Web** target. Make sure that no other options are selected.
+6. Click **Create**.
+
+   ![Kotlin Multiplatform wizard](wasm-kmp-wizard.png){width=600}
 
 ## Run the application
 
-1. In IntelliJ IDEA, open the **Gradle** tool window by selecting **View** | **Tool Windows** | **Gradle**.
+1. Open the **Gradle** tool window by selecting **View** | **Tool Windows** | **Gradle**.
```

Comment:
Could be done separately (if so, please file an issue): 
Normally, there should be a run configuration, and I think it should be a default way we use in the docs.